package.json

```diff
@@ -96,11 +96,11 @@
     "eslint-plugin-promise": "4.3.1",
     "eslint-plugin-standard": "5.0.0",
     "exports-loader": "0.7.0",
-    "husky": "4.3.8",
+    "husky": "^5.2.0",
```

Comment:
Husky v5 includes a breaking change making our current setup unusable, did you test this after you upgraded?